pom.xml

```diff
@@ -130,6 +130,11 @@
 		</extensions>
 	</build>
 	<dependencies>
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-api</artifactId>
+			<version>1.7.1</version>
```

Comment:
❓ any particularly reason why this version of slf4j? Seems like there are newer versions.

Reply:
I mirrored what I saw used in DDF. When testing a jar of these changes in the other system I didn't have any issue. I can try updating it, but I'm not away of any reason other than there are newer releases (up to 1.7.25 now).